iib/web/static/api_v1.yaml

```diff
@@ -862,21 +944,7 @@ components:
                 This determined what level of protection the addition or removal had.
               type: string
               example: 'prod'
-
-    RequestRegenerateBundleOutput:
-      allOf:
-        - $ref: '#/components/schemas/Request'
-        - type: object
-          properties:
-            from_bundle_image:
-              type: string
-              description: The pull specification of the original bundle image that was modified.
-              example: 'quay.io/user/openshift-bundle-example-container:v1'
-            organization:
-              type: string
-              description: The name of the organization the bundle was regenerated for.
-              example: openshift-operators
-    RequestUpdate:
+    RequestUpdate: #TODO shouldn't be just add/rm ??? Is it request or response?
```

Comment:
@yash what request could be updated?

Reply:
This is the schema used in the `requestBody` of the `patch` endpoint. It's a request if I am not wrong